swap_meet/item.py

```diff
@@ -0,0 +1,24 @@
+class Item:
+    def __init__(self, category="", condition = 0):
+        self.category = category
+        self.condition = float(condition)
+
+    def __str__(self):
+        if str(Item):
```

Comment:
`str(Item)` evaluates to something like: <class '__main__.Item'> so under the Python 'truthy' rules this will always evaluate to True.